journal/modernized/27-chapter-26.adoc

```diff
@@ -1,10 +1,10 @@
-== Section XXVI.
+== Chapter XXVI
 
-In 1705, he visits the province of Ulster, accompanied by George Rooke; and in 1706,
-several places, where people were desirous to hear the testimony of truth.
-Meets with abuse at Roscrea.
-Visits Leinster Province Meeting, and the county of Tipperary,
-in company with Thomas Wilson.
+[.chapter-synopsis]
+* In 1705, he visits the province of Ulster, accompanied by George Rooke
+* In 1706, they visit several places, where people were desirous to hear the testimony of truth
```

Comment:
So I used "they" instead of "he" here because it looked like in his journal that they traveled together on and off.

Reply:
Seems good to me.